dpgen/generator/run.py

```diff
@@ -1913,12 +1929,23 @@ def run_md_model_devi(iter_index, jdata, mdata):
 
     model_devi_engine = jdata.get("model_devi_engine", "lammps")
     if model_devi_engine == "lammps":
-        command = f"{{ if [ ! -f dpgen.restart.10000 ]; then {model_devi_exec} -i input.lammps -v restart 0; else {model_devi_exec} -i input.lammps -v restart 1; fi }}"
+        nbeads = jdata["model_devi_jobs"][iter_index].get("nbeads")
+        if nbeads is None:
+            command = f"{{ if [ ! -f dpgen.restart.10000 ]; then {model_devi_exec} -i input.lammps -v restart 0; else {model_devi_exec} -i input.lammps -v restart 1; fi }}"
+        else:
+            command = f"{{ if [ ! -f dpgen.restart.10000 ]; then {model_devi_exec} -p {nbeads}x1 -i input.lammps -v restart 0; else {model_devi_exec} -p {nbeads}x1 -i input.lammps -v restart 1; fi }}"
```

Comment:
The restart filename is `dpgen.restart${ibead}.10000` instead, per the above codes.